src/libraries/System.Reflection.Emit/src/Resources/Strings.resx

```diff
@@ -267,4 +267,22 @@
   <data name="Argument_RedefinedLabel" xml:space="preserve">
     <value>Label defined multiple times.</value>
   </data>
+  <data name="Argument_BadPInvokeMethod" xml:space="preserve">
+    <value>PInvoke methods must be static and native and cannot be abstract.</value>
+  </data>
+  <data name="Argument_BadPInvokeOnInterface" xml:space="preserve">
+    <value>PInvoke methods cannot exist on interfaces.</value>
+  </data>
+  <data name="Argument_MethodRedefined" xml:space="preserve">
+    <value>Method has been already defined.</value>
+  </data>
+  <data name="Argument_GlobalMembersMustBeStatic" xml:space="preserve">
+    <value>Global members must be static</value>
+  </data>
+  <data name="InvalidOperation_GlobalsHaveBeenCreated" xml:space="preserve">
+    <value>Type definition of the global function has been completed.</value>
+  </data>
+  <data name="Argument_BadSizeForData" xml:space="preserve">
+    <value>Data size must be &amp;gt; 1 and &amp;lt; 0x3f0000</value>
```

Comment:
Doesn't a literal `>` and `<` work?

Also should probably add a period.

Reply:
> Doesn't a literal > and < work?

I would expect so, just tested with Runtime implementation it works!
```suggestion
    <value>Data size must be &amp;gt; 1 and &amp;lt; 0x3f0000.</value>
```